### DISCUSSION TITLE
Stress testing / validation?

### DISCUSSION THREAD

**Discussion by Tetsujin13** [daveshap/ACE_Framework#19] [General]
Hi, I suspect that feasibility work will start soon, but as feasibility becomes concluded, are there any plans on creating a general validation document with criteria covering all kinds of uses, or is it up to the user to create their own validation for a specific use?

For example, if creating an AI agent that will be tasked with diplomacy work and be in contact with representatives who act on behalf of a government that knowingly use propaganda and base their reality and understanding on a selection of information that is biased.

Or for example, if creating an AI agent that will be tasked with being an informer or council on the behalf of the society and be in contact with citizens who have opinions on different local/regional policies, cases and affairs, in relation to statements and policies of political parties etc. (i.e. are they economically feasible, based on science, scientifically sound)

**Comment by daveshap** [comment]
I think we'll get to that point further down the road, but the ACE architecture probably won't be ready for that level of work until R4. 

ACE will be more useful for backoffice workers (e.g. not customer facing, only as a parallel aid) for the near term)
